Update README

theodoreleebrant opened the pull request:
Update both README files for the project:

- Remove Vagrant setup
- Include Ubuntu setup
- Remove references to vagrant in .gitignore
- Remove Vagrantfile

theodoreleebrant committed:
Update README (English)

Update the English version of the README file:
- Remove Vagrant setup
- Add Ubuntu manual setup

donjar commented on ENGLISH.md:
```diff
@@ -13,68 +13,99 @@ two members of our organization:
+virtual machine. 
+
+### Prerequisite packages
+For the project to run on your machine, you would need to install several packages:
```
Wrong numbering?

Also, can write the commands for Ubuntu? `sudo apt install software-properties-common` etc.

theodoreleebrant replied:
Edited

donjar commented on ENGLISH.md:
````diff
@@ -13,68 +13,99 @@ two members of our organization:
+    ```bash
+    ruby --version
+    ```
+5. Install `bundler` and `rails`, on the correct version. Currently, rails 4.2.11.1 and bundler 1.17.3 is recommended for the project.
````
Actually, `bundler` is enough

donjar commented on ENGLISH.md:
```diff
@@ -13,68 +13,99 @@ two members of our organization:
+   to install the gems needed. 
+7. Now we move to creating our database. Make sure that `postgresql` is installed.
+    1. Make a file called `database.yml` in the `config` folder. 
+       You can do this by modifying the given `database.yml.default` file in the project.
```
*copying

donjar commented on ENGLISH.md:
````diff
@@ -13,68 +13,99 @@ two members of our organization:
+    4. Run all migration files by `bundle exec rake db:migrate`
+8. The initial setup to run the server has been done. To run the server in test environment, you can run 
+    ```bash
+    bundle exeec rails s -e test
````
```suggestion
    bundle exec rails s -e test
```

Typo?

donjar reviewed (commented):
Generally looks good for me.

- can you make the version for Bahasa Indonesia as well?
- can you remove the Vagrantfile in the directory? And scripts/setup.sh or something as well. Since we're now not using Vagrant anymore.

theodoreleebrant committed:
Update README (Indonesian)

Update the Indonesian version of the README file:
- Remove Vagrant setup
- Add Ubuntu manual setup

theodoreleebrant committed:
Minor edits to README (English)

- Standardize the usage of fullstops
- Clarified postgresql setup steps

theodoreleebrant committed:
Clarify package installation and minor edits

donjar reviewed (approved):
Don't forget to clean up the .gitignore file as well :)

theodoreleebrant committed:
Remove Vagrantfile and update .gitignore
